Emby.Server.Implementations/Localization/Ratings/be.csv

```diff
@@ -1,6 +1,11 @@
-BE-AL,1
-BE-MG6,2
-BE-6,3
-BE-9,5
-BE-12,6
-BE-16,8
+AL,0
+KT,0
+TOUS,0
+MG6,6
+6,6
+9,9
+KNT,12
+12,12
+BE_14,14
```

Comment:
Missed country prefix when they were removed?

Reply:
If I remember correctly `KNT` is not a prefix but a rating and `BE_14` was kept because the country prefix code only used `-` as a delimiter.

Reply:
`KNT` is indeed not a prefix, it's `Kinderen Niet Toegelaten`, Bit odd there is no french equivalent like `AL` (Dutch) has `TOUS` (french), then again Belgium is just a bit weird.

It's only the `BE_14` that seems aweird to me, I could also find no mention of it. There are however age categories 6,9,12,14 and 16. So my guess was BE_ was just a typo and should have been BE- and this also stripped.

But I had a very very hard time finding any form of official standard, closes I found was that the old one was replaced by the dutch one on January 2020 (https://www.brusselstimes.com/belgium/82830/films-broadcast-in-belgium-will-have-a-new-classification-system-from-2020.html) but could find no official government publication.